gulpfile.js

```diff
@@ -47,18 +39,18 @@ gulp.task('clean', function () {
         .pipe(clean());
 });
 
-gulp.task('copy-files', function() {
-    return merge(
-        gulp.src([...orderData, ...dontVulcanizeTheseFiles], {base: './'})
+gulp.task('copy-files', () => 
+    merge2(
+        gulp.src(orderData, {base: './'})
             .pipe(gulp.dest(deployDir)),
         gulp.src([...copyTheseFilesToDist])
             .pipe(replace('INSERT_VERSION', version))
             .pipe(replace('INSERT_SHA', git.short()))
             .pipe(replace('INSERT_BUILD_TIME', new Date().toLocaleString()))
             .pipe(debug('copied files'))
             .pipe(gulp.dest(deployDir))
```

Comment:
Something that has been bugging me is that this corrupts images (e.g. favicon doesn't show up in https://mergemypullrequest.github.io/order-splitter/, but does in https://dumbbillyhardy.github.io/order-splitter/ (because I manually copied it over)), so we'll want to figure out a way fix that sometime (not in this PR dough)

Reply:
Hm. The `manifest.json` is missing (responds with 404). I believe that's where the favicon comes from. I'm able to repeat this bug when I build from the "stable" commit fcc613b from 2017 using node 8 and manually-version-locked bower components. I don't think it's an issue with the latest polymer build.

Reply:
If the only thing we're using gulp for now is deploying, then we can probably just replace it with the [`gh-pages` npm package](https://www.npmjs.com/package/gh-pages).

```bash
npx gh-pages --remote origin -d ./dist
```